src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx

```diff
@@ -198,7 +198,7 @@ export const SavedEndpointsPage = () => {
         heading={{ title: "Saved Endpoints" }}
```

Comment:
I'm not sure how to name "Saved Endpoints". :thinking: "Saved Endpoints and Methods" seems too long. Any ideas, @jeesunikim @janewang @sdfcharles @leighmcculloch ? :pray:

Reply:
@quietbits How about `Saved API Requests`?

Reply:
I like @janewang's suggestion. I'd shorten it to: Saved Requests.

Reply:
Thank you both! :raised_hands: I'll go with "Saved Requests".